packages/blocs/user_poll/lib/bloc/user_poll_bloc.dart

```diff
@@ -37,22 +37,17 @@ class UserPollBloc extends Bloc<UserPollEvent, UserPollState> {
     UserPollEvent event,
   ) async* {
     //
+    yield UserPollLoadingState();
```

Comment:
This makes flutter to always rebuild the widget tree at least twice when already loaded

Reply:
Thank you!